### ISSUE TITLE
Consoldiate histograms

### ISSUE THREAD

**Issue by jberkus** [cncf/devstats.archive#101]
The following five histograms/tables all show counts by github handle with the same filtering controls.  They should be consolidated into a single table.

- Approvers Histogram
- Developers Table
- PR Authors Repository Groups Table
- Reviewers in Repository Groups Table
- Top Commenters in repository groups table

The new table would be called "Developer Activity Counts by Repository Group" and would look exactly like the current Developers Table, only with a "Repository Group" selector added.

@Phillels please approve before @lukaszgryglicki does any work on this.  Thanks!

**Comment by lukaszgryglicki** [comment]
This isnot the same AFAIK, but I will have to check actual SQL's first.
I think some of them use different definitions:
- For example, approver definition counts in all text related tables + labels like lgtm/approve AFAIK
- Reviewers was also defined in some special way AFAIK (PullRequestReviewEvents + labels + texts?)
- PR authors count distinct actors who created PR.

I need to dive into all of them to see their actual logic, unfortunately, they don't have docs yet, otherwise, it would be obvious what each histogram calculates. Initial requirements were provided in google docs about 6+ months ago.

**Comment by jberkus** [comment]
@lukaszgryglicki well, there's two different reviewer charts, one of which I am NOT proposing to consolidate, since its data is differently organized.

**Comment by lukaszgryglicki** [comment]
OK, I'll see the exact data they pull, and will consolidate once approved.

**Comment by lukaszgryglicki** [comment]
Starting very initial research.
This will **NOT** be ready before KubeCon.eu.

**Comment by jberkus** [comment]
Figured.  Do it when you can!

**Comment by lukaszgryglicki** [comment]
Done on test and on prod: https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1
Now I need to fix test coverage.
Closing, please reopen if needed.

**Comment by lukaszgryglicki** [comment]
The funny part is that I wasn't regenerating data, but adding new series.
I've finished all including new dashboard creation, generating data, some fixes, doing the same on the prod etc.
And all this time "InfluxDB" is just dropping data from the old series - no longer used.
Spawned: "delete from /^(approvers_hist|hist_pr_authors|reviewers_hist|top_comm|project_developer)/" command about 3 hours ago and it is stiull running.
Influx really **sucks** on deleting data
@jberkus @dankohn 
I've also created issue on the influx repo (maybe 3 weeks ago).
Previous versions of influx were 1000x or even more faste ron deleting data.
This is really strange for me that simple delete operation takes so long. I know thy're spawning some data cleaner etc etc. But hey, we should have an option to skip this.
Regenerating all Kubernetes data from scratch is faste rthat this single delete.
I had to rewrite many of my devstats logic to avoid deletes back then.
